packages/core/src/lib/stream_manager.ts

```diff
@@ -14,60 +18,108 @@ export class StreamManager {
     public addEventListener: Libp2p["addEventListener"]
   ) {
     this.log = new Logger(`stream-manager:${multicodec}`);
-    this.addEventListener(
-      "peer:update",
-      this.handlePeerUpdateStreamPool.bind(this)
-    );
-    this.getStream = this.getStream.bind(this);
     this.streamPool = new Map();
+
+    this.addEventListener("peer:update", this.handlePeerUpdateStreamPool);
```

Comment:
why change line order?

Reply:
it's a nit -- just for better readability -- moved the `streamPool` definition above attaching an event listener